src/System.Security.Cryptography.Algorithms/tests/ECDiffieHellmanTests.cs

```diff
@@ -35,6 +35,24 @@ public static void ECCurve_ctor()
             }
         }
 
+        [Theory]
+        [InlineData("1.3.132.0.35", "ECDSA_P521", 521)]
+        [InlineData("1.3.132.0.34", "ECDSA_P384", 384)]
+        [InlineData("1.2.840.10045.3.1.7", "ECDSA_P256", 256)]
+        public static void ECCurve_ctor_ECDSA_Algorithm_Name(string oidValue, string oidFriendlyName, int expectedKeySize)
+        {
+            var curveOid = new Oid(oidValue);
+            var ecCurve = ECCurve.CreateFromOid(curveOid);
```

Comment:
Style: `var` is inappropriate here (only allowed for `new`, `as T` and `(T)` hard cast.